composer.json

```diff
@@ -30,5 +30,10 @@
     "psr-4": {
       "WordPress\\Plugin_Check\\": "includes/"
     }
+  },
+  "autoload-dev": {
+    "psr-4": {
+      "WordPress\\Plugin_Check\\Tests\\": "tests/testdata"
```

Comment:
Maybe better to use a `WordPress\Plugin_Check\Test_Data` namespace for this? It's more specific to what the folder indicates should go there. For example, if we introduced test utility classes, they would also semantically fall under the namespace used here, but they would not be the same as test data. So maybe for now a test data namespace works better for this purpose.